pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>com.zendesk</groupId>
     <artifactId>mysql-binlog-connector-java</artifactId>
-    <version>0.29.1</version>
+    <version>0.29.2</version>
```

Comment:
Hi, let's remove this change in the commit.  @osheroff will decide what the version should be based on this and any other changes that end up merged for the release.  It's also just cleaner to leave that task for the release process.

Reply:
done